### DISCUSSION TITLE
We need your feedback ~ 欢迎在这里反馈，制定项目的下一步方向

### DISCUSSION THREAD

**Discussion by binary-husky** [binary-husky/gpt_academic#1671] [Announcements]
已经确定的规划方向：

点击这里查看[已经确定的开发方向](https://github.com/binary-husky/gpt_academic/wiki/ProjectNeedHelp)

**Comment by binary-husky** [comment]
top 1: System Prompt 持久留存

**Comment by rockingapple** [comment]
希望进行精准翻译后的PDF文件能够保留原文的图片和公式

**Reply by binary-husky** [reply]
是的，这个是必须的一环

**Reply by binary-husky** [reply]
公式有点难，图片问题不大

**Reply by rockingapple** [reply]
翻译时，还有个功能希望能加上：就是添加自定义的翻译角色。
例如，之前使用gpt academic进行翻译时，点击“精准翻译PDF论文”后就直接翻译全文了，翻译后的结果不太专业，希望能在翻译前限定翻译者的角色，如果要翻译的是医学论文，就限定翻译者是医学教授，以此类推。

**Comment by rockingapple** [comment]
还有个功能希望能增加进去：
集成一个开源的OCR识别工具，该工具尤其擅长对公式识别，能够把识别的公式转换成Latex格式

**Reply by binary-husky** [reply]
能具体说一下应用场景吗

**Reply by rockingapple** [reply]
之前打算用来识别照片中的数学公式，照片是拍摄的试卷或课本习题集，不过现在我已经找到替代方案了。
如果方便集成OCR就集成吧，若是不方便就不考虑集成这个功能。

**Comment by sumwee** [comment]
可以接入最新的gpt4模型吗

**Reply by binary-husky** [reply]
正在弄，其他意见呢

**Reply by sumwee** [reply]
数据分析之类的插件可以增加吗，对医学或者其他需要数据分析的专业有点帮助

**Comment by Menghuan1918** [comment]
可以优化一下选择插件的界面吗？现在插件多起来了感觉不怎么好选了

**Reply by binary-husky** [reply]
嗯，确实是这样，我想想办法

**Comment by RonghaiHe** [comment]
PDF文档总结部分，能否准确划分PDF论文中的每一部分并进行总结，目前好像只是单纯按字数划分

**Reply by binary-husky** [reply]
是的，目前是的，后面看看怎么改进

**Comment by stevesu2021** [comment]
本地自己serving的大模型要怎么进行配置接入

**Reply by binary-husky** [reply]
wiki里面有

**Comment by jimmy3693** [comment]
能不能编制个说明文档，讲解一下怎样使用，感觉使用上还是有一定门槛的。
